examples/go/vm-reconfig-via-tag/stack.yml

```diff
@@ -0,0 +1,15 @@
+version: 1.0
+provider:
+  name: openfaas
+  gateway: https://veba.yourdomain.com
+functions:
+  vm-reconfig-via-tag-fn:
+    lang: golang-http
```

Comment:
This is the middleware template which allows to return errror/status codes right?

Reply:
Yup. The classic Go template uses `STIO` to communicate, but this watchdog template uses HTTP.

More info here: https://docs.openfaas.com/cli/templates/#classic-vs-of-watchdog-templates